bazel/src/main/java/com/google/api/codegen/bazel/ApiVersionedDir.java

```diff
@@ -33,9 +33,10 @@
       Pattern.compile("(?m)^package\\s+(?<protoPackage>[\\w+\\.]+)\\s*;\\s*$");
   private static final Pattern IMPORTS =
       Pattern.compile("(?m)^import\\s+\"(?<import>[\\w+\\\\./]+)\"\\s*;\\s*$");
-  private static final Pattern PROTO_OPTIONS =
+  // A proto's language package options.
+  private static final Pattern PROTO_LANG_PACKAGE =
       Pattern.compile(
-          "(?m)^option\\s+(?<optName>\\w+)\\s+=\\s+\"(?<optValue>[\\w./;\\\\\\-]+)\"\\s*;\\s*$");
+          "(?m)^option\\s+(?<optName>(java|go|csharp|ruby|php|javascript)_(namespace|package))\\s+=\\s+\"(?<optValue>[\\w./;\\\\\\-]+)\"\\s*;\\s*$");
```

Comment:
Was the less specific regexp failing on practical cases (if yes, can you please give an example?), or is this change mostly to avoid something like that in the future?

Reply:
Yes, it was failing. For Java, it grabbed the first available option, resulting in the wrong value. For example, since java_outer_classname comes first in the [asset](https://github.com/googleapis/googleapis/blob/master/google/cloud/asset/v1/assets.proto) proto, we ended up with the map entry `"java" : "AssetProto"` instead of the expected `"java" : ""com.google.cloud.asset.v1"`. I think this worked for other languages because they generally have only one config option.